packages/mergebot/src/basic.ts

```diff
@@ -53,6 +67,12 @@ export const labelNames = [
 
 export type ApproverKind = "maintainer" | "owner" | "other";
```

Comment:
stupid idea (because the code is already good enough): why not `export type ApproverKind = (typeof approverKindOrder)[number]` ?

but also: numeric enums give you all the features you need with `enum ApproverKind { None, Other, Owner, Maintainer }` to simplify `getMaxApproverKind`.

Reply:
I'll do the former one, but I was avoiding the latter because the ApproverKind is a part of baselines and will end up showing up in the dumped JSON as regular numbers.